client/src/components/FeedbackForm/FeedbackButton.scss

```diff
@@ -0,0 +1,9 @@
+.feedback-button {
+  align-self: center;
+  position: absolute;
+  bottom: 5px;
```

Comment:
### UI Concern

__Known Bug (not your fault)__: Currently, if the viewport has a height so short that bottom of viewport touches nav links, then the nav links will wrap. This is fixed by [PR #175 in `Sidebar.module.scss` via new rules in `.root`](https://github.com/TACC/Frontera-Portal/pull/175/files#r474985976). You may add that specific fix from that PR to this PR, to test whether new code introduces new bug.

__Possible New Bug__: After the known bug is fixed, the `.feedback-button` styles might cause a new problem: if the viewport has a height so short that bottom of viewport touches nav links, then this new feedback link might overlap the bottom-most links in the Sidebar ([flex and position behavior, together, is unique](https://css-tricks.com/flexbox-and-absolute-positioning/)).

### Likely Solution

If bug exists, try relying on `margin` instead of `position`:

```suggestion
  align-self: center;
  margin-top: auto;
  margin-bottom: 5px;
```

The result, for short viewport and with ported `Sidebar` fix, would be no link overlap.

### Reference
- [Flexbox and absolute positioning](https://css-tricks.com/flexbox-and-absolute-positioning/)
- [The peculiar magic of flexbox and auto margins](https://css-tricks.com/the-peculiar-magic-of-flexbox-and-auto-margins/)
- [Flexbox’s Best-Kept Secret](https://hackernoon.com/flexbox-s-best-kept-secret-bd3d892826b6)